lib/surface/components/form/label.ex

```diff
@@ -42,10 +44,11 @@ defmodule Surface.Components.Form.Label do
   def render(assigns) do
     helper_opts = props_to_opts(assigns)
     attr_opts = props_to_attr_opts(assigns, class: get_config(:default_class))
+    event_opts = events_to_opts(assigns) |> opts_to_attrs()
```

Comment:
It's just a matter of taste but I suggest writing it with two pipe operators.

```suggestion
    event_opts = 
       assigns 
       |> events_to_opts()
       |> opts_to_attrs()
```
or
 ```suggestion
     event_opts = assigns |> events_to_opts() |> opts_to_attrs()
```